### DISCUSSION TITLE
How to start rendering quickly in labmda

### DISCUSSION THREAD

**Discussion by a-kifumi** [remotion-dev/remotion#1806] [Q&A]
Rendering a 25-minute video.
The video is 44820 frames. it times out before rendering starts. is there any way to start rendering quickly?

(1/5) [====================] Invoking lambdas 100%. 
(2/5) [ ] Rendering frames (0/44820)
(3/5) [ ] Encoding chunks 0%

Timeout at the above point.

My setting is 
concurrency 1000
memory 3008MB
timeout 900
Other defaults.

The rendering image uses png. This is a Flickering measure in the case of my video.

Memory is set to 3008 MB, the upper limit due to new account restrictions.
In the case of 820 frames, rendering was completed in 220s. This is also mostly due to the time taken to start up the lambda.

Features of the animation are.
・The animation illustrations are used to switch between many mouths and eyes.
・A voice-over is played for each paragraph.
・The video is read from a jpeg, png or webp file for commentary.

The following are some of the features of the video

**Comment by JonnyBurger** [comment]
Hi @yoshisansan! We can debug this, but it requires more information.

Label your delayRenders() by passing in a string and identify which one of those are timing out. Once you know that, place logs in your  Lambda render to figure out if the code is running as you are expecting it to.

Also these techniques will help you debug the render: https://www.remotion.dev/docs/lambda/troubleshooting/debug

If that all does not help, post the relevant snippet that causes the delayRender() timeout!
